core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -169,7 +169,7 @@ public ThreadPool(final Settings settings, final ExecutorBuilder<?>... customBui
         final int genericThreadPoolMax = boundedBy(4 * availableProcessors, 128, 512);
         builders.put(Names.GENERIC, new ScalingExecutorBuilder(Names.GENERIC, 4, genericThreadPoolMax, TimeValue.timeValueSeconds(30)));
         builders.put(Names.INDEX, new FixedExecutorBuilder(settings, Names.INDEX, availableProcessors, 200));
```

Comment:
I think we can migrate `TransportUpdateAction` to redirect to a single item bulk as well (as a follow up). At that point we can rename bulk to index :)